docs/_includes/layouts/base.njk

```diff
@@ -18,10 +18,12 @@
 {% if feedTitle and feedUrl %}
 		<link rel="alternate" href="{{ feedUrl }}" title="{{ feedTitle }}" type="application/atom+xml">
 {% endif %}
+<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@mdi/font@6.5.95/css/materialdesignicons.min.css" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
I'll admit I'm not wild about pulling down 300k just for a single icon, but I guess it'll be cached after the first page load. (Or even before then, if it's on a common CDN.)

Reply:
Good point, it didn't cross my mind when I was trying to get it to work 😅. Another dev raised the same point here - https://github.com/mamezou-tech/eleventy-plugin-code-clipboard/issues/8.
Apparently, both these css links can be removed, and it'll still work (I wasn't using the tooltips anyway). I got it to work, will submit a patch right away!

Reply:
Awesome!